Libraries/Animated/src/Interpolation.js

```diff
@@ -222,13 +222,19 @@ function createInterpolationFromStringOutputRange(
     });
   });
 
+  // rgba requires that the r,g,b are integers.... so we want to round them, but we *dont* want to
+  // round the opacity (4th column).
+  const shouldRound = (/^rgb/).test(outputRange[0]);
+
   return (input) => {
     var i = 0;
     // 'rgba(0, 100, 200, 0)'
     // ->
     // 'rgba(${interpolations[0](input)}, ${interpolations[1](input)}, ...'
     return outputRange[0].replace(stringShapeRegex, () => {
-      return String(interpolations[i++](input));
+      const val : Number = interpolations[i++](input);
```

Comment:
I don't speak eslint/flow, apparently.... does anyone know what I should do to fix this?  Thanks.